### DISCUSSION TITLE
NextJS + Radix-UI doesn't react when Trigger is clicked: [Violation] Added non-passive event listener to a scroll-blocking 'wheel' event.

### DISCUSSION THREAD

**Discussion by dominik-sfl** [radix-ui/primitives#952] [Help]
I copied the `Dropdown` Example Code as well as the `Dialogue` Example Code and am using it in my nextjs site. Works fine in dev. However, when I build the project and click either the Dropdown or the Trigger for the Dialogue, the trigger doesn't fire and nothing really happens. 

Chrome Console tells me that the error is:

> [Violation] Added non-passive event listener to a scroll-blocking 'wheel' event. Consider marking event handler as 'passive' to make the page more responsive. See https://www.chromestatus.com/feature/5745543795965952

Not sure what to make of this though? Any ideas what I could be doing wrong?

**Comment by jjenzz** [comment]
Have you tried creating a fresh NextJS project with the Radix components to rule out any conflicts with other packages in your project? If so, would you mind pushing the repo somewhere public for us to investigate?

**Comment by ashtonlance** [comment]
I'm getting the same error messages from Web Audit too.

**Reply by jjenzz** [reply]
It would be great to see a reproduction somewhere if you are able to provide one please?
